oqsprov/oqs_sig.c

```diff
@@ -860,13 +869,23 @@ static int oqs_sig_verify(void *vpoqs_sigctx, const unsigned char *sig,
                         goto endverify;
                     }
                     if (!strncmp(name, "pss", 3)) {
+                        int salt;
+                        const EVP_MD *pss_mgf1;
+                        if (name[3] == '3') { // pss3072
```

Comment:
Hmm...isn't this a bit risky? Is there no way someone could spec say "pss3196" or so?

Reply:
Yes, if someone added pss3196 it would trigger that condition, but this is only for composite and for composite there is no other pss implementation defined that starts with 3. 
Someone could add a "custom" composite but that wouldn't pass interop.
Regardless, to make it more safe I'll change this condition and add a error in case something different from 3072 or 2048 is passed.